src/components/Colors.js

```diff
@@ -0,0 +1,11 @@
+
+export const Colors = {
+    green: '#03fc7b',
+    red: '#F24444',
+    gold: '#D9C57E',
+    lightgray: '#D3D3D3',
+    black: '#000000',
+    blue: '#0000FF',
+    CornflowerBlue: '#6495ED',
+    white: '#FFFFFF'
+  };
```

Comment:
unused colors.
do you need these green, red gold colors?
you just copy pasted my example